pkg/statistics/handle/storage/read_test.go

```diff
@@ -100,3 +105,53 @@ func TestLoadStats(t *testing.T) {
 	require.Greater(t, float64(cms.TotalCount()+topN.TotalCount())+hg.TotalRowCount(), float64(0))
 	require.True(t, idx.IsFullLoad())
 }
+
+func TestLoadNonExistentIndexStats(t *testing.T) {
+	store, dom := testkit.CreateMockStoreAndDomain(t)
+	tk := testkit.NewTestKit(t, store)
+	tk.MustExec("use test;")
+	tk.MustExec("drop table if exists t;")
```

Comment:
I don't think this is needed in this unit tests, but for realtikv it probably is, so I'm OK of keeping it here as well.